package-lock.json

```diff
@@ -11496,9 +11496,9 @@
       "integrity": "sha1-QRyttXTFoUDTpLGRDUDYDMn0C0A="
     },
     "path-parse": {
-      "version": "1.0.6",
-      "resolved": "https://registry.npmjs.org/path-parse/-/path-parse-1.0.6.tgz",
-      "integrity": "sha512-GSmOT2EbHrINBf9SR7CDELwlJ8AENk3Qn7OikK4nFYAu3Ote2+JYNVvkpAEQm3/TLNEJFD/xZJjzyxg3KBWOzw=="
+      "version": "1.0.7",
```

Comment:
@matthewmascord all our versions in package.json are fixed - is there a missing change to package.json that bumps one of our versions there? may well have misunderstood part of how npm audit fixes stuff but wanted to check

Reply:
As this is a transitive dependency, I'm not sure there's much we can do in package.json to fix this issue. package-lock.json is ultimately the source of truth for the versions used, so everything is pinned there. Having the versions pinned in package.json is a further line of defence in case the package-lock.json is deleted unknowingly.